src/diffusers/models/attention_processor.py

```diff
@@ -1678,3 +1354,287 @@ def forward(self, f, zq):
         norm_f = self.norm_layer(f)
         new_f = norm_f * self.conv_y(zq) + self.conv_b(zq)
         return new_f
+
+
+## Deprecated
```

Comment:
Do you want to directly throw a deprecation message when the class is initialized or would that be too brutal?

Reply:
I thought about this too - think this will be a bit too much though at the moment because it would mean every time we call `unet.attn_processors` we would get a deprecation warning which would be every time we save LoRAs. I'd suggest to only start throwing aggressive deprecation warnings when we do the peft integration